java/gandiva/src/main/cpp/jni_common.cc

```diff
@@ -15,6 +15,7 @@
 // specific language governing permissions and limitations
 // under the License.
 
+#include <arrow/status.h>
```

Comment:
Can you move this include with the other Arrow includes below?

Reply:
done